book/website/ethical-research/ethics-case-study.md

```diff
@@ -0,0 +1,134 @@
+
+### Consent
+>How will you ensure any personal data collected for your research is done so with freely-given informed consent?
+
+**Justification.** The purpose of ensuring consent for the collection, use or reuse of personal data is to respect the autonomy of individuals and operate in line with pertinent legislation.
+It is also an important concept enshrined in law.
+In the General Data Protection Regulation (GDPR), for example, *consent* is defined as:
+
+>“any freely given, specific, informed and unambiguous indication of the data subject’s wishes by which he or she, by a statement or by a clear affirmative action, signifies agreement to the processing of personal data relating to him or her" ([ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/consent/what-is-valid-consent/))
+
+**Responding to this question** entails reflection on how data are sourced -- where they come from and who they pertain to.
+Setting the legal discussions aside, this question requires following one of two approaches:
+1. If employing data that is readily available (perhaps online or provided by a third party), it is important we understand the terms and conditions whereby those data are made available. The point here is to confirm that the data will be used following the purpose for which they were collected. In this respect, a useful subchapter in *The Turing Way* is that of *[Data Licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html?highlight=licen)*, which outlines different ways online data might be held.
+2. If gathering new data through experiments with human research subjects, both personal information sheets and consent forms must be issued. On the one hand, personal information sheets provide research participants with all the information they need to know about taking part in the study. On the other hand, consent forms summarise the personal information sheet and provide a section for the potential participant to record their acceptance of the conditions. The consent form, in particular, should provide contact details for participants to reach out to the research team or institution (depending on internal processes) should they have any queries or concerns about the study in the future. The UK Data Service offer templates of both documents [online](https://www.ukdataservice.ac.uk/manage-data/legal-ethical/consent-data-sharing/consent-forms).
+
+### Privacy and Security
+>How will you ensure that any sensitive information your project uses is kept secure and made accessible only to pertinent members of the research team?
+
+**Justification.** Privacy is a fundamental human right ([TTW](https://github.com/alan-turing-institute/the-turing-way/pull/1394/files?short_path=b95854e#diff-b95854e17574ab3b68f2f89991e4dddf199e01901dc8e235d87e2b0c8980a4d9)).
```

Comment:
This "TTW" link is a bit obscure, it wasn't clear what I was clicking through to.

Reply:
Yes! Okay, I have a question: is it okay for me to link to a raft Turing Way chapter?

Reply:
I _think_ so? @malvikasharan might have a better answer. I think it's nice though because maybe you could annotate the link saying "This is a draft chapter that needs review. If this is something you're interested in doing, please get in touch!"